internal/kuberesource/reader.go

```diff
@@ -32,15 +32,19 @@ func UnmarshalApplyConfigurations(data []byte) ([]any, error) {
 	return result, nil
 }
 
-// YAMLBytesFromFile reads a k8 YAML file and returns a formatting-preserving encoding.
-func YAMLBytesFromFile(yamlPath string) ([]byte, error) {
-	data, err := os.ReadFile(yamlPath)
-	if err != nil {
-		return nil, fmt.Errorf("reading %s: %w", yamlPath, err)
-	}
-	kubeObjs, err := UnmarshalApplyConfigurations(data)
-	if err != nil {
-		return nil, fmt.Errorf("unmarshaling %s: %w", yamlPath, err)
+// YAMLBytesFromFiles reads one or multiple K8s YAML files and returns a formatting-preserving encoding.
```

Comment:
While we're touching this again: I'm not sure if the docstring here is correct - we're not actually preserving formatting, but rather pretty printing, right? The input could even be pure JSON...